index.d.ts

```diff
@@ -82,4 +82,50 @@ for (let index = 1; index <= 6; index++) {
 //=> 6: 2s
 ```
 */
-export default function pThrottle(options: Options): <F extends AnyFunction>(function_: F) => ThrottledFunction<F>;
+export function pThrottleRate(options: RateOptions): <F extends AnyFunction>(function_: F) => ThrottledFunction<F>;
```

Comment:
Since it's a named function, can drop the prefix:
```suggestion
export function throttleRate(options: RateOptions): <F extends AnyFunction>(function_: F) => ThrottledFunction<F>;
```